explorer/client/package.json

```diff
@@ -19,8 +19,8 @@
         "onchange": "^7.1.0",
         "postcss": "^8.4.6",
         "prettier": "2.5.1",
-        "react-scripts": "5.0.1",
         "puppeteer": "^13.5.1",
+        "react-scripts": "5.0.1",
```

Comment:
this order change shouldn't have any effect on the code - could we remove for tidiness' sake?

Reply:
Done